docs/sources/visualizations/histogram-draft.md

```diff
@@ -0,0 +1,107 @@
++++
+title = "Visualize Histograms in Grafana"
+description = "Histogram visualization"
+keywords = ["grafana", "docs", "bar chart", "panel", "prometheus"]
+weight = 605
++++
+---
+
+This section provides detailed information on using histograms and constructing queries to find the number of files based on file size and bucket distribution over time.
+
+# Visualize Prometheus data in Grafana
+
+Histogram visualization provides a graphical representation of the distribution of numerical data. This topic explains how to visualize **Prometheus** data as a histogram in Grafana. For more on this topic, please see [Introduction to histograms and heatmaps](https://grafana.com/docs/grafana/latest/basics/intro-histograms/) and [Prometheus Best Practices: Histograms and summaries](https://prometheus.io/docs/practices/histograms/#apdex-score).
+
+Before you begin, you should have a basic understanding of [Prometheus](https://grafana.com/oss/prometheus/) and Grafana. This documentation will refer to histograms from the perspective of [Grafana 7.0](https://grafana.com/blog/2020/05/18/grafana-v7.0-released-new-plugin-architecture-visualizations-transformations-native-trace-support-and-more/), though later versions of Grafana are also available.
```

Comment:
```suggestion
We assume that you have a basic understanding of [Prometheus](https://grafana.com/oss/prometheus/) and Grafana. 

> **Note:** This topic refers to histograms from the perspective of [Grafana 7.0](https://grafana.com/blog/2020/05/18/grafana-v7.0-released-new-plugin-architecture-visualizations-transformations-native-trace-support-and-more/).
```

Reply:
Fiona had me change "we assume..." to "you shuld have..." from a previous commit. Which one should it be?

Reply:
Jita has more context - please use her suggestion.